examples/encoding/plot_continuous_encoder.py

```diff
@@ -0,0 +1,115 @@
+"""
+=============================
+Regression on continuous data
+=============================
+
+This demonstrates how encoding models can be fit with multiple continuous
+inputs. In this case, a encoding model is fit for one electrode, using the
+values of all other electrodes as inputs. These inputs could also be stimulus
+features, such as spectral features of sound.
```

Comment:
Yeah I def agree, but I wasn't sure what data existed where we could demonstrate this (in a non-rERP form) so I just did electrode -> electrode modeling to demonstrate the API. Maybe if we got some ECoG data to use as an example?